src/engraving/libmscore/measure.cpp

```diff
@@ -4074,6 +4033,9 @@ void Measure::setStretchedWidth(qreal w)
         w = minWidth;
     }
 
+    // New spacing algorithm: this stretching is not necessary anymore.
+
+#if 0
```

Comment:
never use that

If there is something redundant, then just remove it, otherwise the code will turn into a garbage